scripts/emu/simple_dhcp.py

```diff
@@ -32,17 +32,37 @@ def create_profile(self, ns_size, clients_size):
                                       plugs   = {'arp': {},
                                                  'icmp': {},
                                                  'igmp': {}, 
-                                                 'dhcp': {}
-                                                },
+                                                 'dhcp': {
+                                                     'options' : {
```

Comment:
@rzrabi better to have 2 options
1. backward compatibility -- no options provided just ```DHCP: {}`` in case option wan't provided by user
2. in case user provide an option to build the options object.